aspnetcore/native-aot/index.md

```diff
@@ -0,0 +1,183 @@
+---
+title: ASP.NET Core support for .NET native AOT
+author: mitchdenny
+description: Learn about ASP.NET Core support for .NET native AOT
+monikerRange: '>= aspnetcore-8.0'
+ms.author: midenn
+ms.custom: mvc
+ms.date: 04/10/2023
+uid: native-aot/index
+---
+# ASP.NET Core support for .NET native AOT.
+
+ASP.NET Core 8.0 introduces support for [.NET native ahead-of-time (AOT)](/dotnet/core/deploying/native-aot/). Not all ASP.NET Core features are compatible with native AOT at this time. This article covers:
```

Comment:
```suggestion
ASP.NET Core 8.0 introduces support for [.NET native ahead-of-time (AOT)](/dotnet/core/deploying/native-aot/). This article covers:
```


Can we turn this "Not all ASP.NET Core features are compatible with native AOT at this time. " into a yellow warning bar at the top of the article?

cc @Rick-Anderson

Reply:
https://github.com/dotnet/AspNetCore.Docs/pull/28826/files#r1158058640